configs/frags/base.config

```diff
@@ -41,3 +41,7 @@ BR2_ENABLE_LOCALE_WHITELIST="C en_US"
 
 # fs utils
 BR2_PACKAGE_E2FSPROGS=y
+
+# udev needed for generating /dev/disk/by-uuid
+BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
+BR2_PACKAGE_EUDEV=y
```

Comment:
OK I see your last push changed this:
```diff
-BR2_USE_WCHAR=y
+BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
```
Starting some new builds on staging now to confirm.